yarn.lock

```diff
@@ -11035,6 +11752,36 @@ webpack@5.87.0:
     watchpack "^2.4.0"
     webpack-sources "^3.2.3"
 
+webpack@^5, webpack@^5.89.0:
```

Comment:
is this a duplicate?

Reply:
looks like we'd have 2 versions of webpack

Reply:
Maybe it's time to move away from Yarn to either npm or pnpm.

Reply:
> 'yarn dedupe' should solve this.

It didn't

Reply:
try `npx yarn-deduplicate && yarn install`

Reply:
Today I learned :-)